blockchain/mining-token-rewards.md

```diff
@@ -14,7 +14,7 @@ While there's no exact formula for for calculating how much `HNT` you'll earn ov
 
 The Helium blockchain rewards Hotspots for providing wireless coverage and verifying the Helium Network. Hotspots are rewarded in Helium Token \(`HNT`\).
 
-Every [epoch](mining-token-rewards.md), the current consensus group mines a block on the blockchain. In each block, hotspots earn `HNT` for the work they have contributed. Hotspots and other network participants earn `HNT` according to the following distribution:
+Every [epoch](mining-token-rewards.md), the current consensus group mines 30 blocks on the blockchain. In each block, Hotspots earn `HNT` for the work they have contributed according to the following distribution:
```

Comment:
In the second sentence, are we saying hotspots earn HNT in each block, or each epoch? I'm not sure on the technical details of HNT earning, but from the user perspective, they earn HNT per epoch.

Reply:
That's a good question. I'll leave that one to @PharkMillups.

Reply:
Your splitting of hairs is admirable. :) 

HNT is distributed per epoch, though one could go block by block and find the relevant TXs.  I think the exact language should probably he something like: 

`In each block, Hotspots perform various types of work and are awarded according to the following distribution:`